content/get-started/installation/_index.md

```diff
@@ -5,60 +5,88 @@ slug: /get-started/installation
 
 # Install Streamlit
 
-There are multiple ways to set up your development environment and install Streamlit. Read below to
-understand these options. Developing locally with Python installed on your own computer is the most
+There are multiple ways to set up your development environment and install Streamlit.
+Developing locally with Python installed on your own computer is the most
 common scenario.
 
-## Summary for experts
+<Tip>
+
+Try a Streamlit Playground that runs in your browser &mdash; no installation required.
+(Note that this is not how Streamlit is meant to be used, because it has many downsides. That's why it's a
+_playground_!)
+
+<IconLink
+    href="/get-started/installation/streamlit-playground"
+    icon="arrow_forward"
+    label="Instructions for the playground"
+    cssModuleClassName="Indigo"
+    cssModuleIconClassName="IconRight"
+/>
+
+</Tip>
+
+## Summary for experienced Python developers
+
+1. To set up your Python environment and test your installation, execute the following terminal commands:
 
-1. Set up your Python development environment.
-2. Run:
    ```bash
    pip install streamlit
-   ```
-3. Validate the installation by running our Hello app:
-   ```bash
    streamlit hello
    ```
-4. Jump to our [Basic concepts](/get-started/fundamentals/main-concepts).
-
-## Installation steps for the rest of us
 
-<TileContainer layout="list">
+1. Jump to our [Basic concepts](/get-started/fundamentals/main-concepts).
 
-<RefCard href="/get-started/installation/command-line" size="half">
+## Install Streamlit on your machine
 
-<h5>Option 1: I'm comfortable with the command line</h5>
+### Option 1: I like the command line
 
 Install Streamlit on your own machine using tools like `venv` and `pip`.
 
-</RefCard>
-
-<RefCard href="/get-started/installation/anaconda-distribution" size="half">
+<IconLink
+    link="/get-started/installation/command-line"
+    icon="arrow_forward"
+    label="Instructions for the command line"
+    cssModuleClassName="Orange"
+    cssModuleIconClassName="IconRight"
+/>
 
-<h5>Option 2: I prefer a graphical interface</h5>
+### Option 2: I prefer a graphical interface
 
 Install Streamlit using the Anaconda Distribution graphical user interface. This is also the best
 approach if you're on Windows or don't have Python set up.
 
-</RefCard>
+<IconLink
+    link="/get-started/installation/anaconda-distribution"
+    icon="arrow_forward"
+    label="Instructions for Anaconda Distribution"
+    cssModuleClassName="Orange"
+    cssModuleIconClassName="IconRight"
+/>
 
-<RefCard href="/get-started/installation/community-cloud" size="half">
+## Create an app in the cloud
 
-<h5>Option 3: I'd rather use a cloud-based environment</h5>
+### Option 1: I want a free cloud environment
```

Comment:
Non-blocking: I'd love to make option 1 feel more like the community option, and option 2 like the pro option.